analysis/estimate/notebooks/run-estimate.Rmd

```diff
@@ -0,0 +1,289 @@
+---
+title: "Run ESTIMATE on bulk and pseudobulk samples"
+author: "Stephanie J. Spielman"
+date: "`r Sys.Date()`"
+output:
+  html_notebook:
+    toc: true
+    toc_depth: 3
+    code_folding: hide
+---
+
+The purpose of this notebook is to run ESTIMATE on bulk (counts and TPM) and pseudobulk samples and compare results to one another.
+
+## Setup
+
+
+```{r setup}
+renv::load()
+
+library(ggplot2)
+library(tidyestimate)
+theme_set(theme_bw())
+```
+
+### Paths
+
+```{r paths}
+data_dir <- here::here("analysis", "estimate", "data")
+sample_metadata_file <- here::here("s3_files", "scpca-sample-metadata.tsv")
+library_metadata_file <- here::here("s3_files", "scpca-library-metadata.tsv")
+expression_files <- list.files(
+  data_dir, 
+  pattern = "_expression\\.rds$"
+) |>
+  purrr::set_names(
+    \(x) {stringr::str_split_i(x, pattern = "_", i = 1)}
+  )
+
+output_file <- here::here("analysis", "estimate", "results", "estimate_results.tsv")
+```
+
+### Read input data
+
+```{r}
+sample_metadata_df <- readr::read_tsv(sample_metadata_file, show_col_types = FALSE)
+library_metadata_df <- readr::read_tsv(library_metadata_file, show_col_types = FALSE)
+expression_data <- expression_files |>
+  purrr::map(
+    \(filename) {
+      readr::read_rds(file.path(data_dir, filename))
+})
+```
+
+## Run ESTIMATE
+
+We will now run ESTIMATE on all three types of counts for samples in each project:
+
+* bulk counts
+* bulk TPM
+* pseudobulk counts
```

Comment:
Can you include brief descriptions of what these are? I know we've been using them in other analyses, but I think it would be helpful context when just looking at this notebook. Mainly I just want to be explicit that bulk counts and TPM come from the bulk RNA-seq data and pseudobulk counts come from the single-cell RNA seq data.